lib/testutils/httpmultibin/httpmultibin.go

```diff
@@ -247,9 +248,9 @@ func NewHTTPMultiBin(t testing.TB) *HTTPMultiBin {
 		KeepAlive: 10 * time.Second,
 		DualStack: true,
 	})
-	dialer.Hosts = map[string]net.IP{
-		httpDomain:  httpIP,
-		httpsDomain: httpsIP,
+	dialer.Hosts = map[string]lib.IPPort{
+		httpDomain:  lib.NewIPPort(httpIP),
+		httpsDomain: lib.NewIPPort(httpsIP),
```

Comment:
Possibly outside of this PR as I don't think it's worth it, but with the new "hosts" we can drop the `_PORT` and `_URL` varaints